### PR TITLE
feat: add Arbitrum chain

### PR DESCRIPTION
**Pull request by mohandast52**
### Added support for Arbitrum and Arbitrum Sepolia

- Mint and list

https://github.com/valory-xyz/autonolas-registry-frontend/assets/22061815/2b736713-bb1f-4255-a0e3-1461a495eab4

- Step 1

https://github.com/valory-xyz/autonolas-registry-frontend/assets/22061815/5112b063-4dd4-4b3a-8385-298aa005a912

- Step 2

https://github.com/valory-xyz/autonolas-registry-frontend/assets/22061815/b19b0e7c-6e95-42b0-af65-72f3b0e34877

- Step 3

https://github.com/valory-xyz/autonolas-registry-frontend/assets/22061815/b4413629-76c3-452c-800e-6976f8d81f72

- Operator whitelist

https://github.com/valory-xyz/autonolas-registry-frontend/assets/22061815/0cd9000e-1c1b-45aa-8231-32597f86a6a9

## Types of changes

What types of changes does your code introduce to `autonolas-registry-frontend`?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [x] My pull request is targeted against the main branch (left side), and I've started my branch from our main.
- [ ] I've ensured that linting and unit tests pass locally with my changes, and CI also passes successfully.
- [ ] I've included tests to validate the effectiveness of my fix or to demonstrate that my feature functions correctly.
- [ ] I've verified that my changes do not reduce code coverage.
- [x] I've added any necessary documentation as needed.
- [x] I've confirmed that any dependent changes have been merged and published in downstream modules.
- [ ] If the onload page involves contract interaction, I've ensured it is client-side rendered using next/dynamic.
- [ ] I've updated next.config.js header to include the new URL for secure app interaction

### CODE REVIEW

**Review by kupermind — DISMISSED** [review]

**Review by oaksprout — APPROVED** [review]

File: common-util/Contracts/addresses.jsx [hunk 1]
```diff
@@ -123,6 +143,8 @@ export const FALLBACK_HANDLER = {
   10200: '0xf48f2B2d2a534e402487b3ee7C18c33Aec0Fe5e4',
   80001: '0xf48f2B2d2a534e402487b3ee7C18c33Aec0Fe5e4',
   31337: '0x0000000000000000000000000000000000000000',
+  42161: '0xf48f2B2d2a534e402487b3ee7C18c33Aec0Fe5e4',
```
Comment (DavidMinarsch) [inline common-util/Contracts/addresses.jsx:146]: There is nothing at this address: https://arbiscan.io/address/0xf48f2B2d2a534e402487b3ee7C18c33Aec0Fe5e4

File: common-util/Contracts/addresses.jsx [hunk 2]
```diff
@@ -110,6 +128,8 @@ export const safeMultiSend = {
   10200: ['0x40A2aCCbd92BCA938b02010E17A5b8929b49130D'],
   80001: ['0x40A2aCCbd92BCA938b02010E17A5b8929b49130D'],
   31337: ['0x9d4454B023096f34B160D6B654540c56A1F81688'],
+  42161: ['0x9d4454B023096f34B160D6B654540c56A1F81688'],
```
Comment (DavidMinarsch) [inline common-util/Contracts/addresses.jsx:131]: nothing here: https://arbiscan.io/address/0x9d4454B023096f34B160D6B654540c56A1F81688

Comment (DavidMinarsch) [inline common-util/Contracts/addresses.jsx:131]:
```suggestion
  42161: ['0x40A2aCCbd92BCA938b02010E17A5b8929b49130D'],
```
as per https://github.com/safe-global/safe-deployments/blob/main/src/assets/v1.3.0/multi_send_call_only.json

File: common-util/Contracts/addresses.jsx [hunk 3]
```diff
@@ -110,6 +128,8 @@ export const safeMultiSend = {
...
+  421614: ['0x9d4454B023096f34B160D6B654540c56A1F81688'],
```
Comment (DavidMinarsch) [inline common-util/Contracts/addresses.jsx:132]:
```suggestion
  421614: ['0x40A2aCCbd92BCA938b02010E17A5b8929b49130D'],
```
as per same link

File: common-util/Contracts/addresses.jsx (hunk 1 above)
Comment (DavidMinarsch) [reply common-util/Contracts/addresses.jsx:146]: no issue here

**Review by DavidMinarsch — APPROVED** [review]

**Review by oaksprout — APPROVED** [review]

**Pull request merged**